content/blog/deploying-mongodb-application-on-hpe-greenlake-for-containers.md

```diff
@@ -0,0 +1,948 @@
+      infoMessage: '',
+      electionTime: Timestamp({ t: 1669297790, i: 2 }),
+      electionDate: ISODate("2022-11-24T13:49:50.000Z"),
+      configVersion: 6,
+      configTerm: 1,
+      self: true,
+      lastHeartbeatMessage: ''
+    },
+    {
+      _id: 1,
+      name: '10.192.3.59:27017',
+      health: 1,
+      state: 2,
+      stateStr: 'SECONDARY',
+      uptime: 311,
+      optime: { ts: Timestamp({ t: 1669298880, i: 1 }), t: Long("1") },
+      optimeDurable: { ts: Timestamp({ t: 1669298880, i: 1 }), t: Long("1") },
+      optimeDate: ISODate("2022-11-24T14:08:00.000Z"),
+      optimeDurableDate: ISODate("2022-11-24T14:08:00.000Z"),
+      lastAppliedWallTime: ISODate("2022-11-24T14:08:00.869Z"),
+      lastDurableWallTime: ISODate("2022-11-24T14:08:00.869Z"),
+      lastHeartbeat: ISODate("2022-11-24T14:08:05.090Z"),
+      lastHeartbeatRecv: ISODate("2022-11-24T14:08:05.094Z"),
+      pingMs: Long("0"),
+      lastHeartbeatMessage: '',
+      syncSourceHost: '10.192.3.58:27017',
+      syncSourceId: 0,
+      infoMessage: '',
+      configVersion: 6,
+      configTerm: 1
+    },
+    {
+      _id: 2,
+      name: '10.192.4.208:27017',
+      health: 1,
+      state: 2,
+      stateStr: 'SECONDARY',
+      uptime: 56,
+      optime: { ts: Timestamp({ t: 1669298880, i: 1 }), t: Long("1") },
+      optimeDurable: { ts: Timestamp({ t: 1669298880, i: 1 }), t: Long("1") },
+      optimeDate: ISODate("2022-11-24T14:08:00.000Z"),
+      optimeDurableDate: ISODate("2022-11-24T14:08:00.000Z"),
+      lastAppliedWallTime: ISODate("2022-11-24T14:08:00.869Z"),
+      lastDurableWallTime: ISODate("2022-11-24T14:08:00.869Z"),
+      lastHeartbeat: ISODate("2022-11-24T14:08:05.104Z"),
+      lastHeartbeatRecv: ISODate("2022-11-24T14:08:03.615Z"),
+      pingMs: Long("0"),
+      lastHeartbeatMessage: '',
+      syncSourceHost: '10.192.3.59:27017',
+      syncSourceId: 1,
+      infoMessage: '',
+      configVersion: 6,
+      configTerm: 1
+    }
+  ],
+  ok: 1,
+  '$clusterTime': {
+    clusterTime: Timestamp({ t: 1669298880, i: 1 }),
+    signature: {
+      hash: Binary(Buffer.from("0000000000000000000000000000000000000000", "hex"), 0),
+      keyId: Long("0")
+    }
+  },
+  operationTime: Timestamp({ t: 1669298880, i: 1 })
+}
+rs0 [direct: primary] test>
+```
+
+## HPE GreenLake for Containers: Data Science with MongoDB deployed on Kubernetes cluster
+
+Below steps illustrate interactive Jupyter notebook adding records and querying MongoDB deployed on 'hpe' Kubernetes cluster.
+
+![](/img/image-13.png)
+
+Create a Database instance 'glcaasmongodemo' and sample collection 'glsamplecollection' and add document records in database.
+
+![](/img/image-14.png)
+
+Add multiple documents to sample collection
+
+![](/img/image-15.png)
+
+Perform query operation on sample collection
+
+![](/img/image-16.png)
+
+## HPE GreenLake for Containers: Demo Summary
+
+Github Repository Hosting Demo Code: https://github.com/cxteamtrials/caas-trials-content
```

Comment:
Recommend rephrasing this to "You can find the GitHub repository that hosts demo code here." and hotlink "here" or "GitHub repository" with the link instead of writing it out.

Also, you need a little more of a summary. Maybe something to the effect of:
"We hope that this blog post has provided you with enough information for you to get started deploying a containerized, stateful MongoDB application using HPE GreenLake for Containers. To view more articles and tutorials on the use of HPE GreenLake for Containers, refer to the HPE Developer Community blog. (hotlink to developer.hpe.com) For even more information, you can... (and maybe you can refer them to some documentation site you may have. Remember to use hotlinks for these)

Reply:
done